### PR TITLE
export images with different file-endings

### LINKED ISSUE #907: export movie-logo results in jpg
**Describe the bug**
currently the save logo files are all `png`s which is perfectly fine. but when exporting everything to html file the file ending is `jpg`.

**To Reproduce**
1. Go to Export
2. Click on Export, but make sure the theme exports the logo as well.
3. Open folder and inspect files

**Expected behavior**
The file-content is still a valid png, so just the file-ending is wrong.

**MediaElch Version:**
 - [x] 2.6.2 stable

**Operating System:**
 - [x] Linux (distribution: arch)

### PR DESCRIPTION
**Pull request by Stunkymonkey**
fix #907

### CODE REVIEW

File: src/export/SimpleEngine.cpp [hunk 1]
```diff
@@ -562,9 +562,6 @@ void SimpleEngine::replaceImages(QString& m,
 
 bool SimpleEngine::saveImageForType(const QString& type, const QSize& size, QString& destFile, const Movie* movie)
 {
-    destFile = "movie_images/"
-               + QString("%1-%2_%3x%4.jpg").arg(movie->movieId()).arg(type).arg(size.width()).arg(size.height());
-
```
Comment (bugwelle) [inline src/export/SimpleEngine.cpp]: Wow, how did I miss that when I refactored the exporter...?
Thanks! :)

**Comment by bugwelle** [comment]
Looks good to me. Thank you for the contribution! Very much appreciated. 😃 
The code looks fine but I still want to manually test your PR. I'll have time in a few days. I hope that is OK.

**Comment by Stunkymonkey** [comment]
sure. no problem.

**Comment by bugwelle** [comment]
Works as expected. Thanks. :-)
I just noticed that some images are stored without any contents... (size is 0 Bytes). That's unrelated to this PR but yet another issue.

(🇩🇪  Grüße aus Heidelberg)

**Pull request merged**